client/src/containers/EditTrip/__tests__/index.spec.js

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { create } from 'react-test-renderer';
+import { shallow } from 'enzyme';
+ 
+import EditTrip from './../../EditTrip';
+import { GET_TRIP } from './../../../constants/action-names';
```

Comment:
Not sure if I need a real action name and correct payload if anyway I am mocking the action, but at least it works now